Channels/Integration/03_ConfigureLogging.md

```diff
@@ -0,0 +1,89 @@
+[!!User interface Logging](../UserInterface/to_be_completed)
+
+# Configure the logging settings
+
+Configure in the logging settings the log levels of messages to be stored and their retention time.
+
+
+## Define the minimum log level
+
+Define the minimum log level from which all messages will be stored with the selected and higher level.
```

Comment:
```suggestion
Define the minimum log level for the messages to be stored. For all messages with the selected and higher level a log will be created.
```
Mein Vorschlag ist nicht wirklich schön ;), kannst du sicher verbessern, aber ich finde den ursprunglichen Satz schwer zu verstehen und redundant (minimun - selected and higher level).